sycl/unittests/xpti_trace/QueueIDCheck.cpp

```diff
@@ -136,7 +136,7 @@ TEST_F(QueueID, QueueCreationAndKernelWithDeps) {
   checkTaskBeginEnd(QueueIDSTr);
 }
 
-TEST_F(QueueID, QueueCreationUSMOperations) {
+TEST_F(QueueID, DISABLED_QueueCreationUSMOperations) {
```

Comment:
It would be nice to have internal tracker for this, so that we can reenable it back.

Reply:
Done: https://github.com/intel/llvm/issues/12963